livingdoc-converters/src/main/kotlin/org/livingdoc/converters/collection/ListConverter.kt

```diff
@@ -0,0 +1,14 @@
+package org.livingdoc.converters.collection
+
+open class ListConverter : AbstractCollectionConverter<List<Any>>() {
+
+    override fun convertToTarget(collection: List<Any>): List<Any> {
+        return collection
+    }
+
+    override fun canConvertTo(targetType: Class<*>?): Boolean {
+        val isJavaObjectType = List::class.javaObjectType == targetType
```

Comment:
`javaObjectType` is only relevant for Java classes which have primitive variants (`boolean`, `int` etc.), use `java` instead